server/controllers/auth.js

```diff
@@ -26,31 +27,37 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   const user = await User.create({
     name,
     email,
-    password
+    password,
   });
 
   if (user) {
     await Profile.create({
       userId: user._id,
-      name
+      name,
+    });
+
+    await Availability.create({
+      petSitterId: user._id,
+      schedules: [],
+      activeSchedule: null,
```

Comment:
Not sure about creating a new availability for a user. This should happen if they are registering as a merchant but if they are a regular user we won't have availability.

Let's remove this for now while we figure things out with signing up and converting to a merchant.